internal/storage/localdir.go

```diff
@@ -52,25 +57,25 @@ func (s LocalDir) Delete(catalog string) error {
 }
 
 func (s LocalDir) ContentURL(catalog string) string {
```

Comment:
This is kinda weird. We have a v1-specific storage interface, but it is the thing telling us what our base URL is? Seems like the easiest thing to do right now, but we should make an issue about this and look into some refactoring as a follow-up.

```suggestion
func (s LocalDir) BaseURL(catalog string) string {
```